### ISSUE TITLE
Standartize comments

### ISSUE THREAD

**Issue by Piterden** [apicase/apicase#1]
For class:
```js
/**
 * Class Example
 *
 * @package       Package Name
 * @author        Ryan Thompson <ryan@cms.com>
 */
class Example extends Parent {
  // ...
}
```
For methods:
```js
/**
 * Run after a record is created.
 *
 * @param  object    $entry
 * @return undefined
 */
created (entry) {
  // ...
}
```
If inside code:
```js
// You can't delete yourself.
if (this.id === auth.id) {
  return false
}
```

**Comment by Kelin2025** [comment]
Resolved in 212984f  
I'll add it to master after 0.5 release

**Comment by Piterden** [comment]
Thnx)
